app/experimenter/kinto/tasks.py

```diff
@@ -162,60 +165,67 @@ def nimbus_push_experiment_to_kinto(experiment_id):
 
 
 @app.task
-@metrics.timer_decorator("end_experiment_in_kinto")
-def nimbus_end_experiment_in_kinto(experiment_id):
+@metrics.timer_decorator("pause_experiment_in_kinto")
+def nimbus_pause_experiment_in_kinto(experiment_id):
```

Comment:
Swapping the order of the pause & edit tasks here makes the diff confusing to interpret. I think I see what's up, though

Reply:
Bleargh sorry, i was trying to reorganize it so it always goes

launch
pause
end

it made the diff much uglier than it should've been.  Maybe next time I'll just do reordering in a separate commit so it's easier to see the logical changes.